src/node_i18n.cc

```diff
@@ -91,6 +91,8 @@ using v8::Int32;
 using v8::Isolate;
 using v8::Local;
 using v8::MaybeLocal;
+using v8::NewStringType;
+using v8::Local;
```

Comment:
Can you remove this? It is already defined on line 92

Reply:
A good tip is to sort the list, that way duplicates are easy to spot (some editors will even have a "sort and remove duplicated" action).

Reply:
Ohh, we even lint for that:
```
using statements aren't sorted in 'src/node_i18n.cc'.
	Line 93: Actual: v8::MaybeLocal, Expected: v8::Local
	Line 94: Actual: v8::NewStringType, Expected: v8::MaybeLocal
	Line 95: Actual: v8::Local, Expected: v8::NewStringType
```